### ISSUE TITLE
Cannot scroll in hover documentation popup with f2 when it's already focused

### ISSUE THREAD

**Issue by lesquoyb** [gama-platform/gama#204]
**Describe the bug**
On the popup that appears when hovering a keyword, we can see "Press 'F2' for focus", but if you start by clicking or selecting text on that popup, pressing F2 actually doesn't do anything. To be able to scroll in the popup, the only way is to click on the bottom part (where "Press 'F2' for focus" is written)
![image](https://github.com/gama-platform/gama/assets/6374469/cac43213-71b7-4e33-a1cb-7cc055ad7b1d)

**To Reproduce**
1. hover some keyword
2. when the popup appears, click on it or select some text inside
3. Try to press F2 to have the scroll 
4. See that nothing happens

**Expected behavior**
F2 focuses the popup window and we can scroll in the documentation, or alternatively the text could be "click to focus"

**Additional context**
I guess the problem is that I'm clicking on the popup so the F2 event is not caught by the editor but by the popup.

**Comment by AlexisDrogoul** [comment]
Seems like Windows does not behave the same. On macOS, "focus" is automatically granted when I select something in the popup (or even move the mouse inside). Like here: 

![screencast 2024-09-03 13-40-29](https://github.com/user-attachments/assets/fb86bba1-ceea-4fd2-83b2-767f89e1d4ac)

Can you show what is happening on Windows ?

**Comment by lesquoyb** [comment]
On Windows it stays in the first state (the one before you hover over it on macos) that you can see in my screen capture until you click on the part of the view where it's written "press f2" and at that moment only it goes to the second state shown in your gif.

**Comment by AlexisDrogoul** [comment]
Is it behaving in the same way in Eclipse ?

**Comment by hqnghi88** [comment]
No, on my win, in eclipse, when hover an element, when the cursor moved inside the popup, it switch to "F2 clicked" state, means there is VHscrollers, but there is a delay from the moment the cursor moved into to F2 state.

**Comment by hqnghi88** [comment]
Precisely, in GAMA, the focus is catch by the bottom bar of the popup dialog (where it shown "Press F2..") whatosever press or select text in popup dialog, just move the cursor there will always activate the F2 state. But in eclipse, the focus is the whole popup dialog
